pkg/config/application.go

```diff
@@ -58,6 +58,9 @@ type GenericApplicationSpec struct {
 	EventWatcher []EventWatcherConfig `json:"eventWatcher"`
 	// Configuration for drift detection
 	DriftDetection *DriftDetection `json:"driftDetection"`
+	// NOTE: Do not use the field, only added here to avoid unmarshalling error.
```

Comment:
To clarify what's unmarshalling error
```suggestion
	// NOTE: Do not use the field in pipedv0, only added here to avoid unmarshalling error for compatibility with pipedv1.
```